tests/test_earthspy.py

```diff
@@ -12,7 +12,7 @@
 import pandas as pd
 import requests
 import sentinelhub as shb
-
+import json
```

Comment:
it's important that you test your code before pushing. Below you use `os` but it's not imported at the top.

Reply:
os is actually imported in line 11